packages/@yoda/multimedia/mediaplayer.js

```diff
@@ -6,7 +6,22 @@ var EventEmitter = require('events').EventEmitter
 var AudioManager = require('@yoda/audio').AudioManager
 var delegate = require('@yoda/util/delegate')
 
+var endoscope = require('@yoda/endoscope')
+var MediaPlayerEventMetric = new endoscope.Enum('yodaos:multimedia:media_player_event', {
+  labels: [
+    'id',
+    'url'
+  ],
+  states: [
+    'prepared',
+    'playbackcomplete',
+    'playing',
+    'stopped'
+  ]
+})
```

Comment:
I'm unsure if adding the metrics on library looks well, this makes this library depends on the flora.

Reply:
Maybe an example registry is useful and that could be specified by the environment variable?

Reply:
`@yoda/endoscope` doesn't depends on flora. Instead, only registries that added flora exporter do depends on flora, which is totally upon how you use it.